sdk-tests/src/test/java/io/dapr/it/tracing/http/Service.java

```diff
@@ -13,13 +13,17 @@
 
 package io.dapr.it.tracing.http;
 
+import io.dapr.it.DaprRunConfig;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 
 /**
  * Service for subscriber.
  */
+@DaprRunConfig(
+        enableAppHealthCheck = true
```

Comment:
This is a must to avoid a race condition between invoking a service and it being available.